src/widgetComponents/Matrix1dInput/Row/index.js

```diff
@@ -30,20 +30,31 @@ const defaultProps = {
     readOnly: false,
 };
 
+const orientationStyleMaps = {
+    horizontal: styles.horizontalRow,
+    vertical: styles.verticalRow,
+    pivoted: styles.pivotedRow,
+};
+
 export default class Matrix1dRow extends React.PureComponent {
     static propTypes = propTypes;
     static defaultProps = defaultProps;
 
     static cellKeyExtractor = d => d.key;
 
     rendererParams = (key, data) => ({
-        children: data.value,
+        children: (
```

Comment:
We should avoid doing this. This will create an object in every render (removing the advantage of using List component)